aii-core/src/main/perl/Shellfe.pm

```diff
@@ -229,19 +231,23 @@ sub app_options
          HELP    => 'where to store lock files',
          DEFAULT => "/var/lock/quattor" },
 
+       { NAME    => 'parallel=i',
+         HELP    => 'run commands on hosts in parallel ',
+         DEFAULT => 0 },
+
+       { NAME    => 'logpid',
+         HELP    => "Add process ID to the log messages (disabled by default, unless parallel is > 1)",
+         DEFAULT => undef },
+
          # Options for osinstall plug-ins
        { NAME    => 'osinstalldir=s',
          HELP    => 'Directory where Kickstart files will be installed',
          DEFAULT => '/osinstall/ks' },
 
          # Options for DISCOVERY plug-ins
-         { NAME => 'dhcpconf=s',
-           HELP => 'name of dhcp configuration file',
-           DEFAULT => '/etc/dhcpd.conf' },
```

Comment:
It should work as long as the option in your config file is processed... but I am not sure this is the case... we need to ensure that the default is not changed from previous version, else any site relying on the previous default will have a problem...